README.md

```diff
@@ -147,36 +146,33 @@ There is a [docker-compose.yml](https://github.com/mattermost/mattermost-plugin-
 
 You can login with these credentials:
 
-- Username: `nagiosadmin`
-- Password: `nagios`
+-   Username: `nagiosadmin`
```

Comment:
There are lots of unneeded whitespace changes in the most recent commit. Can you please revert them?

Reply:
I figured that unneeded whitespace has been added by Prettier formatter, which I have removed in the latest commit.